<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addwindow</class>
 <widget class="QMainWindow" name="addwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding new data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Добавление новых данных</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Затраты:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Дата:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Наименование:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="NameEdit"/>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QComboBox" name="dayBox">
                <item>
                 <property name="text">
                  <string>01</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>02</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>03</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>04</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>05</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>06</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>07</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>08</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>09</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>13</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>17</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>18</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>20</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>21</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>22</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>23</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>24</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>25</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>26</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>27</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>28</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>29</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>30</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>31</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="monthBox">
                <item>
                 <property name="text">
                  <string>01</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>02</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>03</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>04</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>05</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>06</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>07</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>08</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>09</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="yearBox">
                <item>
                 <property name="text">
                  <string>2021</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2022</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2023</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2024</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2025</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2026</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Цена:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="PriceEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Количество:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="AmountEdit"/>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Тип:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="typeBox">
              <item>
               <property name="text">
                <string>Продукты</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Связь</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Бытовая химия</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Лекарство</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Одежда</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Другое</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Приход:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Дата:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QComboBox" name="dayBox_2">
              <item>
               <property name="text">
                <string>01</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>02</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>03</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>04</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>05</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>06</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>07</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>08</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>09</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>16</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>17</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>18</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>21</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>22</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>23</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>26</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>27</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>28</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>29</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>31</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="monthBox_2">
              <item>
               <property name="text">
                <string>01</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>02</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>03</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>04</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>05</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>06</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>07</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>08</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>09</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>12</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="yearBox_2">
              <item>
               <property name="text">
                <string>2021</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2022</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2023</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2024</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2025</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2026</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Сумма:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Описание:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="totalLine"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="aboutLine"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
